requirements.txt

```diff
@@ -95,5 +95,6 @@ urllib3==1.22
 wcwidth==0.1.7
 webencodings==0.5.1
 Werkzeug==0.14.1
+wget==3.2
```

Comment:
`requests` module have only a monkey patch for FTP servers, so had to resort to wget which works well, and also displays a progress bar while downloading a file from the server (in this case **GEO FTP server**).

Reply:
Using inbuilt ftplib instead now.